.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Can you explain the benefits of husky?

Reply:
It's just a simple way to manage git hooks. With [several million downloads a week](https://www.npmjs.com/package/husky), it's seems like a pretty standard tool these days to solve this problem.